### PR TITLE
improved color contrast for file menus

### LINKED ISSUE #4000: A11y_AzureNotebooks_Jupyter_MenuBar_WAT: Color Contrast is less than 4.5:1 for Menus(File, Edit, View....etc).
**Environment Details:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Operating System:Windows10

**ReproSteps:**
1. Login with the appropriate credentials to azure notebook account.
2. Navigate to any of the file names present in the samples-->>clickin the file name-->Welcome page     opens with menubar.
3.  Open WAT Tool.
4. Open Color Contrast Analyser.
5. Verify Color Contrast for Menus(File, Edit, View....etc).

**Actual:**
Color Contrast is less than 4.5:1 for Menus(File, Edit, View....etc).

**Expected:**
Color Contrast should be >= 4.5:1 for Menus(File, Edit, View....etc).

**User Impact:**
If ColorContrast is <4.5:1 than Low Vision Users face difficulty to See the Menus.

**Attachment:**
a11y_azurenotebooks_jupyter_menubar_wat

### PR DESCRIPTION
**Pull request by bhansa**
Udpdated the navbar background color and navbar link color Fixes #4000

### CODE REVIEW

**Comment by bhansa** [comment]
please review 
cc: @willingc

File: notebook/static/notebook/less/variables.less [hunk 1]
```diff
@@ -26,3 +26,6 @@
 @cell_padding:                  6px;
 @cell_border_width:             1px;
 @cell_left_border_width:        5px;
+
+@navbar-default-bg:               darken(#f8f8f8, 10%);
+@navbar-default-link-color:       darken(#777, 13%);
```
Comment (minrk) [inline notebook/static/notebook/less/variables.less]: Instead of darkening both, why not just darken the text?

Comment (bhansa) [reply notebook/static/notebook/less/variables.less]: removed the variable to darken navbar

**Comment by bhansa** [comment]
Any update on this one?

**Comment by bhansa** [comment]
cc: @minrk

**Pull request merged**